internal/storage/v2/clickhouse/schema/schema.sql

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE IF NOT EXISTS spans (
+    id String,
+    trace_id String,
+    trace_state String,
```

Comment:
is this the `tracestate` header? can we add comments? why is it so high at the top?